Gems/Atom/RPI/Code/Source/RPI.Public/RenderPipeline.cpp

```diff
@@ -265,6 +265,16 @@ namespace AZ
             }
         }
 
+        void RenderPipeline::SetDefaultStereoscopicViewFromEntity(EntityId entityId, uint32_t viewIndex)
+        {
+            ViewPtr cameraView;
+            ViewProviderBus::EventResult(cameraView, entityId, &ViewProvider::GetStereoscopicView, viewIndex);
```

Comment:
I am wondering if it's supposed to get the two views from one entity. If so, is it necessary to add the viewIndex for the parameter? 

If the viewIndex is necessary, I think we should make the ViewType enum public instead have it hiding in camera component.